### PR TITLE
Update Maven installation guide

### PR DESCRIPTION
**Pull request by akphi**
Update Maven installation guide to follow the latest `sdlc`, `engine`, and `studio` codebase.

We still need to update the config templates from these respective repositories to make this PR effective. Track:
https://github.com/finos/legend-sdlc/pull/129
https://github.com/finos/legend-engine/pull/171
https://github.com/finos/legend-studio/pull/133

### CODE REVIEW

File: installers/docker-compose/legend-with-remote-gitlab/README.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ docker-compose --env-file ~/legend-temp/.env --profile all up --detach
```
Comment (epsstan) [inline installers/docker-compose/legend-with-remote-gitlab/README.md:33]: Can you please move the docker/docker-compose installer changes to a different PR ?
They can be merged independent of the other documentation changes in this PR.

**Review by epsstan — CHANGES_REQUESTED** [review]

File: installers/docker-compose/legend-with-remote-gitlab/README.md (hunk 1 above)
Comment (akphi) [reply installers/docker-compose/legend-with-remote-gitlab/README.md]: Updated!

File: docs/installation/maven-install.md [hunk 2]
```diff
@@ -8,128 +8,73 @@ The Legend platform has five components. The minimum required to run Legend are
...
+  - Enable the "Confidential" check box
+  - Enable these scopes: openid, profile, api
+  - Redirect URI: Add the followings:
+
+```sh
+http://localhost:7070/api/auth/callback
+http://localhost:7070/api/pac4j/login/callback
+```
+
 - Java Development Kit (JDK) 8 or later.
```
Comment (epsstan) [inline docs/installation/maven-install.md:34]: Each project's development setup instructions mention JDK 11. We should change to be consistent.
(We build with JDK11 but target JDK8).

Comment (akphi) [reply docs/installation/maven-install.md]: Updated to 11 !

**Review by epsstan — APPROVED** [review]

**Pull request merged**